mac/cmake/1qt.cmake.in

```diff
@@ -254,8 +269,14 @@ ENDIF ()
 
 IF (@QSCINTILLA_LIBRARY@ MATCHES ".*libqscintilla2.dylib")
     MESSAGE (STATUS "Copying QScintilla2 library and updating library paths...")
+<<<<<<< HEAD
     EXECUTE_PROCESS (COMMAND ditto ${QARCHS} "@QSCINTILLA_LIBRARY@" "${QLIBDIR}/libqscintilla2.dylib")
     GET_INSTALL_NAME ("@QSCINTILLA_LIBRARY@" "libqscintilla2" QSCI_CHG)
+=======
+    EXECUTE_PROCESS (COMMAND ditto ${QARCHS} "@QSCINTILLA_LIBRARY@" "${QLIBDIR}/${QSCI_LIB}.dylib")
+    EXECUTE_PROCESS (COMMAND install_name_tool -delete_rpath "@QT_LIBRARY_DIR@" "${QLIBDIR}/${QSCI_LIB}.dylib")
+    GET_INSTALL_NAME ("@QSCINTILLA_LIBRARY@" "${QSCI_LIB}" QSCI_CHG)
+>>>>>>> daaeeb40fc... strip Qt rpath when bundling
```

Comment:
Merge conflict...

Reply:
Well, the other 4 commits from my PR are missing, again, and this last commit has context from changes in a previous commit.

Reply:
missing commits happened with 3.10 backport #35209